tests/integration/test_registry.py

```diff
@@ -1,166 +1,114 @@
-import os
 import uuid
-from unittest import TestCase
 
-from parameterized import parameterized
+import pytest
 
-from spark8t.domain import Defaults, PropertyFile, ServiceAccount
-from spark8t.services import (
-    AbstractServiceAccountRegistry,
-    K8sServiceAccountRegistry,
-    KubeInterface,
-)
-from tests import integration_test
+from spark8t.domain import PropertyFile, ServiceAccount
 
 
-class TestRegistry(TestCase):
-    kube_interface: KubeInterface
-    defaults = Defaults(
-        dict(os.environ) | {"KUBECONFIG": f"{os.environ['HOME']}/.kube/config"}
+@pytest.mark.parametrize(
+    "namespace, user",
+    [("default-namespace", "spark"), ("spark-namespace", "spark-user")],
```

Comment:
[note] I'm much wondering that we should not be using the same namespace across multiple tests (except `default`), as it prevents us from being able to run our tests simultaneously.